### PR TITLE
Add parameters name to arguments in file_path creation functions

### PR DESCRIPTION
**Pull request by kbeker**
Opened again because I have accidentally closed PR

### CODE REVIEW

**Comment by kbeker** [comment]
@dybi already pushed, ready for reviev

File: concent_api/utils/helpers.py [hunk 1]
```diff
@@ -113,18 +113,22 @@ def sign_message(golem_message, priv_key):
 
 
 def get_storage_file_path(category, subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None and category is not None
     return f'blender/{category}/{task_id}/{task_id}.{subtask_id}.zip'
 
 
 def get_storage_result_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
```
Comment (dybi) [inline concent_api/utils/helpers.py]: those asserts are not needed, it is checked in `get_storage_file_path`

File: concent_api/utils/helpers.py [hunk 2]
```diff
@@ -113,18 +113,22 @@ def sign_message(golem_message, priv_key):
...
     return get_storage_file_path('result', subtask_id, task_id)
 
 
 def get_storage_scene_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
```
Comment (dybi) [inline concent_api/utils/helpers.py]: those asserts are not needed, it is checked in `get_storage_file_path`

File: concent_api/utils/helpers.py [hunk 3]
```diff
@@ -113,18 +113,22 @@ def sign_message(golem_message, priv_key):
...
     return get_storage_file_path('scene', subtask_id, task_id)
 
 
 def get_storage_source_file_path(subtask_id, task_id):
+    assert subtask_id is not None and task_id is not None
```
Comment (dybi) [inline concent_api/utils/helpers.py]: those asserts are not needed, it is checked in `get_storage_file_path`

File: concent_api/utils/helpers.py (hunk 1 above)
Comment (kbeker) [reply concent_api/utils/helpers.py]: Done

File: concent_api/utils/helpers.py (hunk 2 above)
Comment (kbeker) [reply concent_api/utils/helpers.py]: Done

File: concent_api/utils/helpers.py (hunk 3 above)
Comment (kbeker) [reply concent_api/utils/helpers.py]: Done

**Pull request merged**